Formula/yosys.rb

```diff
@@ -0,0 +1,77 @@
+class Yosys < Formula
+  desc "Framework for Verilog RTL synthesis"
+  homepage "http://www.clifford.at/yosys/"
+  url "https://github.com/cliffordwolf/yosys/archive/yosys-0.7.tar.gz"
+  sha256 "3df986d0c6bf20b78193456e11c660f2ad935cc126537c2dc5726e78896d6e6e"
+
+  option "without-abc", "Build without support for ABC"
+  option "without-tcl", "Build without support for Tcl language bindings"
+
+  depends_on "python3"
+  depends_on "libffi" => :recommended
+  depends_on "readline" => :recommended
+  depends_on "pkg-config" => :build
+  depends_on "bison" => :build
+
+  # Prevent the makefile from trying to automatically locate homebrew and macports libraries
+  patch :DATA
```

Comment:
There is a recent upstream change that addresses this issue. I'll make the comment clearer and add a link.

https://github.com/cliffordwolf/yosys/pull/303

With the upstream changes, brew or macports libraries search paths are automatically added only if `brew` or `port` are found on $PATH. So I think it won't be a problem when building in the sandbox.

In general, is there a recommended way for build processes to automatically locate libraries from homebrew or other third-party package managers? Is it discouraged?